<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<section xmlns="http://docbook.org/ns/docbook" version="5.0" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="_classfc_calc_exception" xml:lang="en-US">
<title>fcCalcException Class Reference</title>
<indexterm><primary>fcCalcException</primary></indexterm>
<para>
<computeroutput>#include &lt;fcexceptions.hpp&gt;</computeroutput>
</para>
Inheritance diagram for fcCalcException:<para>
    <informalfigure>
        <mediaobject>
            <imageobject>
                <imagedata width="50%" align="center" valign="middle" scalefit="0" fileref="../../d7/d9b/classfc_calc_exception__inherit__graph.svg"></imagedata>
            </imageobject>
        </mediaobject>
    </informalfigure>
</para>
Collaboration diagram for fcCalcException:<para>
    <informalfigure>
        <mediaobject>
            <imageobject>
                <imagedata width="50%" align="center" valign="middle" scalefit="0" fileref="../../da/dc4/classfc_calc_exception__coll__graph.svg"></imagedata>
            </imageobject>
        </mediaobject>
    </informalfigure>
</para>
<simplesect>
    <title>Public Member Functions    </title>
        <itemizedlist>
            <listitem><para><link linkend="_classfc_calc_exception_1ae56902d3f8c5ed62de60b4c909492971">fcCalcException</link> (const char *whyException=&quot;calculation gets errors&quot;)</para>
</listitem>
        </itemizedlist>
</simplesect>
Public Member Functions inherited from <link linkend="_classfc_runtime_exception">fcRuntimeException</link>        <itemizedlist>
            <listitem><para><link linkend="_classfc_runtime_exception_1a264ae184e45fbc84cc58952ba062c5f5">fcRuntimeException</link> (const char *whyException=&quot;runtime exception occured&quot;)</para>
</listitem>
            <listitem><para>const char * <link linkend="_classfc_runtime_exception_1ac9faee48241584efdec3f0e2955623b1">what</link> ()</para>
</listitem>
        </itemizedlist>
<section>
<title>Detailed Description</title>
<para>
Definition at line <link linkend="_fcexceptions_8hpp_source_1l00034">34</link> of file <link linkend="_fcexceptions_8hpp_source">fcexceptions.hpp</link>.</para>
</section>
<section>
<title>Constructor &amp; Destructor Documentation</title>
<anchor xml:id="_classfc_calc_exception_1ae56902d3f8c5ed62de60b4c909492971"/><section>
    <title>fcCalcException()</title>
<indexterm><primary>fcCalcException</primary><secondary>fcCalcException</secondary></indexterm>
<indexterm><primary>fcCalcException</primary><secondary>fcCalcException</secondary></indexterm>
<para><computeroutput>fcCalcException::fcCalcException (const char * whyException = <computeroutput>&quot;calculation&#160;gets&#160;errors&quot;</computeroutput>
)<computeroutput>[inline]</computeroutput></computeroutput></para><para>
Definition at line <link linkend="_fcexceptions_8hpp_source_1l00037">37</link> of file <link linkend="_fcexceptions_8hpp_source">fcexceptions.hpp</link>.</para>
<programlisting linenumbering="unnumbered">00037 :&#32;<link linkend="_classfc_runtime_exception">fcRuntimeException</link>(whyException)&#32;{};
</programlisting></section>
</section>
</section>
